python/cudf_polars/cudf_polars/dsl/expressions/string.py

```diff
@@ -379,6 +385,73 @@ def do_evaluate(
                 ),
                 dtype=self.dtype,
             )
+        elif self.name is StringFunction.Name.SplitN:
+            n = self.options[0]
+            child, expr = self.children
+            column = child.evaluate(df, context=context)
+            if n == 1:
+                plc_column = plc.Column(
+                    self.dtype.plc,
+                    column.obj.size(),
+                    None,
+                    None,
+                    0,
+                    column.obj.offset(),
+                    [column.obj],
+                )
+            else:
+                assert isinstance(expr, Literal)
+                by = plc.Scalar.from_py(expr.value, expr.dtype.plc)
+                plc_table = plc.strings.split.split.split(
```

Comment:
Minor nit: That's a lot of "split"... maybe we should consider using some of the functions from the subpackage to shorten some of these.